src/styles/card.scss

```diff
@@ -0,0 +1,114 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+:root {
+  font-family: $font;
+}
+
+body {
+  margin: 0;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card {
```

Comment:
The `.card` block element should not have a set height of `100%` unless it is explicitly required for a specific reason. This can lead to unexpected behavior when the content of the card changes. Consider using `min-height` or allowing the card to grow with its content.